### ISSUE TITLE
Onboarding - Password checkbox text goes beyond the margin on small screens

### ISSUE THREAD

**Issue by Francesca-G** [status-im/status-react#21192]
# Design Feedback 2.30

## [Onboarding] Password checkbox text goes beyond the margin on small screens
On small screens the Password disclaimer box doesn't wrap as it should. The text goes beyond the right margin and I'm afraid that on even smaller screens it isn't readable.

Review on Figma [here](https://www.figma.com/design/oznddnBnDbLlLQIaACYNuj?node-id=2249-7147#937845415)

#### Expected behavior

<img width="200" alt="Screenshot 2024-09-06 alle 13 27 09" src="https://github.com/user-attachments/assets/9fdfe682-afb1-4a3f-9cfa-4ba4d77a4947">

Design for small screens [here](https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=11465-91131&t=Gq9ltLaRmdgVuE6c-1)

#### Actual behavior

<img width="200" alt="Screenshot 2024-09-06 alle 13 26 11" src="https://github.com/user-attachments/assets/6784f15f-e8ff-453f-ba5a-b65065a1f0b8">

### Reproduction

1. Create a profile on a small screen device
2. Go through the Onboarding flow until the Create password screen

### Additional Information

- Operating System: Android 14
- Devices in the screenshots: Realme 10

**Comment by ilmotta** [comment]
FYI: The designs in this issue are outdated. The checkbox won't be used anymore.

<img src="https://github.com/user-attachments/assets/0b0c847f-bfde-4fc0-9f77-816681417c0f" width="600" />

https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=15075-44178&node-type=instance&m=dev

**Comment by ulisesmac** [comment]
@ilmotta Can we close this then?

**Comment by ilmotta** [comment]
@ulisesmac I think we can't close without proving the bug doesn't exist anymore. Maybe the text won't wrap correctly in small screens because maybe the bug wasn't caused by the existence of the checkbox.

**Comment by churik** [comment]
@Parveshdhull please, let us know if that's still the issue in a new onboarding process, thanks!

**Comment by seanstrom** [comment]
It seems we no longer have any confirmation checkboxes in the Onboarding flow, and we've also implemented the updated designs mentioned above: https://github.com/status-im/status-mobile/issues/21192#issuecomment-2455581143

So perhaps we can close this issue now?

cc @ilmotta @churik @Francesca-G

**Comment by Parveshdhull** [comment]
>  I think we can't close without proving the bug doesn't exist anymore. Maybe the text won't wrap correctly in small screens because maybe the bug wasn't caused by the existence of the checkbox.

You're right, the bug doesn't seems related to the checkbox. I've encountered this issue before (I will share the link if I find it). In my case, the issue was with the React Native text view itself (probably, I only have vague memory of it) . It added a small extra margin as available space, more than what was actually available. While I can't speak to how it behaves on smaller screens, but the screenshots shared in the issue description suggest that the text is extending slightly beyond the expected margin.

It might still work well on smaller devices, though.

We can probably close this issue for now and revisit it if it resurfaces.

**Comment by Parveshdhull** [comment]
> (I will share the link if I find it)

I was unable to find the link or reproduce the issue, but I remember the text overflowing by only around two characters, possibly while working on https://github.com/status-im/status-mobile/issues/13793.

UPD: Related [so question](https://stackoverflow.com/questions/36284453/react-native-text-going-off-my-screen-refusing-to-wrap-what-to-do) and https://github.com/facebook/react-native/issues/1438

**Comment by seanstrom** [comment]
@Parveshdhull I think this might be caused from the text component not having a style of `:flex 1`. I just pushed up a fix for similar glitch mentioned here: https://github.com/status-im/status-mobile/issues/21911 and it seemed that the text needed the flex style before it would adjust to the container element layout.

**Comment by Horupa-Olena** [comment]
@Parveshdhull  @seanstrom @ilmotta @churik Hi. It look good for my Iphone SE (1334x750-pixel resolution at 326 ppi) and build from 27.01.25. So I propose close this bug. What do you think? 

![Image](https://github.com/user-attachments/assets/d987aaeb-6a0c-4374-b2c5-cc57c3648c11)
